packages/generic-application/lib/server.js

```diff
@@ -0,0 +1,186 @@
+'use strict'
+
+import { printSchema } from 'graphql'
```

Comment:
Unfortunately, `graphql` has a Java-like dependency strategy that requires that _only one version of GraphQL to be loaded_. I think the best option would be to actually load this from the app itself.

Reply:
So we would require the app to export the schema as a string? It works fine to me. We probably just have to document it.

Reply:
We could also try to require `graphql` from the app folder with `createRequire` if they don't.